README.md

```diff
@@ -104,9 +104,45 @@ you want what behaviour. This is the recommended approach.
 
 The default timeout for contacting the server is 1 second, a request will be repeated once in case a timeout occurs, with circuit breakers applied.
 
-#### Create own authorization
+#### Create own authorization / using your own servers
 You can do so by subclassing OAuth2Action and `override def validateToken`.
 
+#### How to mock values in tests?
+Just create this in your project:
+
+```scala
+import scala.concurrent.duration._
+import scala.concurrent.Future
+import com.typesafe.config.Config
+
+import org.zalando.hutmann.authentication._
+trait Auth {
```

Comment:
The main reason is that I think you will most probably derive your own autoRejectBehaviour, and then nevertheless need to re-create the trait. I could include one, just unsure if it is of any use.

However, I am thinking if it does not make more sense to runtime-inject the behaviour starting with Play 2.5, since when you do not compile-time inject - but that is subject to another PR.